CHANGELOG.md

```diff
@@ -21,6 +21,12 @@ This page contains the release notes for AMD ROCm Software.
 
 ### New in this release:
 
+#### OS and GPU support changes
+
+* AMD Instinct™ MI300A and MI300X support has been added
```

Comment:
>AMD Instinct™ MI300A and MI300X support has been added

Not sure what this means. We should considering separating OS from GPU.

Remove 'has been'/passive and is redundant. How about,

Add a one-line description to the section:

The following operating systems are now supported:

- RHEL 9.3
- RHEL 8.9